server/src/main/scala/io/delta/standalone/internal/DeltaSharedTableLoader.scala

```diff
@@ -180,6 +181,160 @@ class DeltaSharedTable(
     snapshot.version
   }
 
+  private def getResponseProtocol(p: Protocol, responseFormat: String): Object = {
+    if (responseFormat == "delta") {
+      model.DeltaProtocol(p.minReaderVersion).wrap
+    } else {
+      model.Protocol(p.minReaderVersion).wrap
+    }
+  }
+
+  private def getResponseMetadata(
```

Comment:
consider adding headers for these functions.

Reply:
Added response header, is that what you are asking for?

Reply:
added